tcms/rpc/tests/test_testexecution.py

```diff
@@ -58,20 +58,20 @@ def test_create_with_no_required_fields(self):
                 self.rpc_client.TestExecution.create(value)
 
     def test_create_with_required_fields(self):
-        tcr = self.rpc_client.TestExecution.create({
+        execution = self.rpc_client.TestExecution.create({
```

Comment:
Next time when you want to rename something outside the main scope do so in a separate commit.

Previous code review talks about renaming variables only in the new tests.